.ci/magic-modules/create-pr.sh

```diff
@@ -29,7 +26,7 @@ $(git log -1 --pretty=%B)
 
 <!-- This change is generated by MagicModules. -->
 EOF
-git checkout -t "$(git branch -a --contains "$(git rev-parse HEAD)" | grep -v "detached" | xargs echo -n)"
+git checkout -b "$BRANCH_NAME"
```

Comment:
How can a checkout work with the same commit on both MM and TF repos? (I mean checkout on lines 20 and 29)

Reply:
Never mind :)